run/test/specs/disappear_after_send_groups.spec.ts

```diff
@@ -23,21 +24,27 @@ async function disappearAfterSendGroups(platform: SupportedPlatformsType) {
   // Create contact between User A and User B
   await createGroup(platform, device1, userA, device2, userB, device3, userC, testGroupName);
 
-  await setDisappearingMessage(platform, device1, ['Group', `Disappear after send option`]);
-  // await runOnlyOnIOS(platform, () => device1.navigateBack(platform));
-  // Check control message
-  await console.log(`Control message not working on Android: ignoring`);
-  // await Promise.all([
-  //   device1.disappearingControlMessage(
-  //     `You set messages to disappear ${time} after they have been ${controlMode}.`
-  //   ),
-  //   device2.disappearingControlMessage(
-  //     `${userA.userName} has set messages to disappear ${time} after they have been ${controlMode}.`
-  //   ),
-  //   device3.disappearingControlMessage(
-  //     `${userA.userName} has set messages to disappear ${time} after they have been ${controlMode}.`
-  //   ),
-  // ]);
+  await setDisappearingMessage(platform, device1, ['Group', `Disappear after send option`, time]);
+  // Get correct control message for You setting disappearing messages
+  const disappearingMessagesSetYou = localize('disappearingMessagesSetYou')
+    .withArgs({ time, disappearing_messages_type: controlMode })
+    .strip()
+    .toString();
+  // Get correct control message for userA setting disappearing messages
+  const disappearingMessagesSetControl = localize('disappearingMessagesSet')
+    .withArgs({ name: userA.userName, time, disappearing_messages_type: controlMode })
+    .strip()
+    .toString();
+  // Check control message is correct on device 2
+  await Promise.all([
+    device1.disappearingControlMessage(disappearingMessagesSetYou as DisappearingControlMessage),
+    device2.disappearingControlMessage(
+      disappearingMessagesSetControl as DisappearingControlMessage
+    ),
+    device3.disappearingControlMessage(
+      disappearingMessagesSetControl as DisappearingControlMessage
```

Comment:
you really shouldn't use `as` for those. Too easy to break.

Reply:
What should I use instead? 
I had to typify them otherwise I got the red line of doom...

Reply:
This probably requires further discussion with @Bilb. I can't really help here.

Reply:
if we get rid of the strings coming from crowdin and use plain strings we won't have that issue anymore.